### PR TITLE
Mark Razor.Extensions.Version1_X as shipoob and exclude

### PR DESCRIPTION
**Pull request by ajaybhargavb**
@muratg @rynowak

### CODE REVIEW

File: packages/packages.csv [hunk 1]
```diff
@@ -79,7 +79,7 @@ Microsoft.AspNetCore.Mvc.Formatters.Xml,ship,include
 Microsoft.AspNetCore.Mvc.Localization,ship,include
 Microsoft.AspNetCore.Mvc.Razor,ship,include
 Microsoft.AspNetCore.Mvc.Razor.Extensions,ship,include
-Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X,ship,include
+Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X,shipoob,exclude
```
Comment (pranavkm) [inline packages/packages.csv:82]: Are we actually shipping this as a package? I thought it was meant for VS only.

Comment (pranavkm) [reply packages/packages.csv:82]: Guess I misunderstood what we intend to do with this package. :shipit: 

Make sure you don't have to make a change in MetaPackages (cc @JunTaoLuo )

Comment (JunTaoLuo) [reply packages/packages.csv:82]: Seems like this says exclude, so no changes to MetaPackage is required.

Comment (muratg) [reply packages/packages.csv:82]: @rynowak should this be `shipoob` or `noship`?

Comment (rynowak) [reply packages/packages.csv:82]: `shipoob` - this package and a few others from Razor ship in VS **only**

**Pull request merged**